kairon/shared/utils.py

```
@@ -501,6 +501,24 @@
         logger.info(f"deleting data from path: {path}")
         shutil.rmtree(path, ignore_errors)
 
+    @staticmethod
+    def remove_file_path(path: Text, default_path: Text = None):
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)